src/content/docs/telemetry-data-platform/manage-data/query-limits.mdx

```diff
@@ -204,5 +204,55 @@ Attributes on `newrelic.resourceConsumption.impact`
 - `Impact` - A count of what is happening when resource has exceeded set limit, i.e dropped requests. 
 - `consumingAccountId` - The New Relic account where the resource is being consumed.
 
- 
+## Set alerts on resource metrics
+
+While building a dashboard to see all your limits is handy, being able to automate it is even better. You can set alerts on your limit metrics to provide updates on limits changes.
+
+<Callout variant="tip">
+Because we currently only have metrics on 1 minute time windows, setting TimeWindow = 1 minute, will cover them all. Eventually, we make more metrics available, you might want to set separate alerts for limits that are enforced by different time windows.
+</Callout>
+
+You can use the following NRQL queries to create alerts. Learn about creating alerts with [NRQL queries here](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions/#create).
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id=""
+    title="Limits faceted by LimitName and scoped by Timewindow"
+  >
+```
```

Comment:
I'd give a little intro/context here just so it's not just 'expand the clamshell and just see the query'. For example, we could say something about 'here's one example of...' to give context about how these are examples and you might theoretically be using a slightly different query; or maybe this is just the correct way to query these things and it's very proscriptive; either way be nice to know.